script.js

```diff
@@ -1,28 +1,32 @@
 $(document).ready(function(){
 
-
-
-
   var lightSeq = [];
   var turn = 0;
-  var attempts = 0;
-
+  var attempt = 1;
+  var strictMode = false;
+  var colorIndex = {Red:1,Green:2,Yellow:3,Blue:4};
+//audio
+  var audioCtx = new (window.AudioContext || window.webkitAudioContext)();
+  var oscillator = audioCtx.createOscillator();
+  var gainNode = audioCtx.createGain();
+  oscillator.start();
+  oscillator.type = 'sine';
+  gainNode.connect(audioCtx.destination);
 
 
   $("#Start").click(function(){
-    lightSeq = [];
-    turn = 2;
-    for(var i=0; i<20; i++){
-      lightSeq.push(Math.floor(Math.random()*4));
-    }
+    generateSeq();
     computerPhase();
+    $("#Start").text('Restart');
+  });
 
+  $("#Strict").click(function(){
+    strictModeToggle($(this));
   });
 
   //player phase
   const playerPhase = function(){
```

Comment:
To better organize this code, I would consider declaring this standalone function outside of the `ready` function.